fdbt-aws/cloudformation/service/eu-west-2/fargate.yml

```diff
@@ -106,7 +90,7 @@ Resources:
         Fn::ImportValue: !Sub ${Stage}:EcsSiteTaskRoleArn
       ContainerDefinitions:
         - Name: !Sub ${ProductName}-site-${Stage}
-          Cpu: 1024
+          Cpu: 512
```

Comment:
I thought we were getting more CPU? We seem to have gone down here?

Reply:
We're going from 1x1024 to 3x512